--- Makefile.orig	2017-05-16 10:09:09.906628779 +0200
+++ Makefile	2017-05-16 11:17:33.412931143 +0200
@@ -29,14 +29,11 @@
 all: $(PROG) $(TEST_PROG)
 
 # Adjust $(HTSDIR) to point to your top-level htslib directory
-HTSDIR = htslib-1.4.1
-include $(HTSDIR)/htslib.mk
-include $(HTSDIR)/htslib_static.mk
-HTSLIB = $(HTSDIR)/libhts.a
-BGZIP  = $(HTSDIR)/bgzip
-TABIX  = $(HTSDIR)/tabix
-HTSLIB_LDFLAGS = $(HTSLIB_static_LDFLAGS)
-HTSLIB_LIBS = $(HTSLIB_static_LIBS)
+HTSDIR = $(EBROOTHTSLIB)/include
+HTSLIB = $(EBROOTHTSLIB)/lib/libhts.a
+BGZIP  = $(EBROOTHTSLIB)/bin/bgzip
+TABIX  = $(EBROOTHTSLIB)/bin/tabix
+HTSLIB_LIBS= -llzma -lbz2 -lcurl -lcrypto -L${EBROOTZLIB}/lib 
 
 CC       = gcc
 CPPFLAGS =
@@ -73,7 +70,7 @@
 ifdef USE_GPL
     EXTRA_CPPFLAGS += -DUSE_GPL
     OBJS += polysomy.o peakfit.o
-    GSL_LIBS = -lgsl -lcblas
+    GSL_LIBS = -L${EBROOTGSL}/lib -lgsl -lcblas
 endif
 
 prefix      = /usr/local
