--- rsem-1.2.15/Makefile.orig	2014-07-15 15:17:41.732690403 +0200
+++ rsem-1.2.15/Makefile    	2014-07-15 15:17:54.174904035 +0200
@@ -1,4 +1,4 @@
-CC = g++
+CC = g++ $(CFLAGS)
 CFLAGS = -Wall -c -I.
 COFLAGS = -Wall -O3 -ffast-math -c -I.
 PROGRAMS = rsem-extract-reference-transcripts rsem-synthesis-reference-transcripts rsem-preref rsem-parse-alignments rsem-build-read-index rsem-run-em rsem-tbam2gbam rsem-run-gibbs rsem-calculate-credibility-intervals rsem-simulate-reads rsem-bam2wig rsem-get-unique rsem-bam2readdepth rsem-sam-validator rsem-scan-for-paired-end-reads
