--- augustus-3.2.2/common.mk.orig	2016-07-08 15:36:55.664775082 +0200
+++ augustus-3.2.2/common.mk	2016-07-08 15:36:51.427712406 +0200
@@ -7,11 +7,11 @@
 ZIPINPUT = true
 
 # uncomment this line to enable comparative gene finding (requires compiler which supports C++11 standard)
-# COMPGENEPRED = true
+COMPGENEPRED = true
 
 # uncomment this line when you need MySQL access to sequences (most users don't)
 #MYSQL = true
 
 # uncomment this line to enable access to SQLite databases that store
 # file offsets of sequence data in flat files and hints
-# SQLITE = true
+SQLITE = true
